ERCS/erc-7558.md

```diff
@@ -0,0 +1,69 @@
+---
+eip: 7558
+title: Minimal Lockable Range NFTs
+description: A minimal interface for soulbinding multiple ERC-721 NFTs
+author: Hans Bergren (@0xCLARITY)
+discussions-to: https://ethereum-magicians.org/t/erc-7558-minimal-lockable-range-nfts/16668
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-11-15
+requires: 165, 721
+---
+
+## Abstract
+
+This proposal extends [ERC-721](./eip-721.md). It proposes a minimal interface to make both individual tokens and a contiguous range of tokenIds soulbound (locked).
```

Comment:
```suggestion
This proposal extends [ERC-721](./eip-721.md) and [ERC-5192](./eip-5192.md). It proposes a minimal interface to make both individual tokens and a contiguous range of tokenIds locked.
```

Adding 5192 and removing "soulbound" since these can be unlocked.